package.json

```diff
@@ -61,8 +61,7 @@
     "eslint": "3.9.1",
     "eslint-plugin-import-order": "2.1.4",
     "ink-docstrap": "1.3.2",
-    "istanbul": "1.1.0-alpha.1",
-    "jsdoc": "4.0.4",
+    "jsdoc": "3.6.11",
```

Comment:
looks like `jsdoc` has been downgraded here, is it intended?

Reply:
yes, it is intended, bumping to v4 breaks the ink-docstrap jsdoc plugin

Reply:
Works for me, thanks for the clarification.